third_party/boost/BUILD.gn

```diff
@@ -0,0 +1,17 @@
+config("boost_config") {
+  visibility = [
+    "//brave/components/brave_wallet/browser",
```

Comment:
This one can be removed, only brave_wallet/common is needed in this list. Will remove it tomorrow.

Reply:
Fixed in https://github.com/brave/brave-core/pull/12384/commits/bd1f5cef70889fff915f8c73058b6163fc989121